cmd/namegen/main.go

```diff
@@ -20,7 +20,7 @@ func main() {
 	flag.Parse()
 	
 	if *list == true {
-		fmt.Print("Available name lists: \nenglish \nspanish \ngerman \nthai\n\n")
+		fmt.Print("Available name lists: \nEnglish \nSpanish \nGerman \nThai \nJapanese\n\n")
```

Comment:
These should be all lowercase.

Reply:
Oh right that makes sense. I guess passing origin thorugh strings.ToLower would add an unnecessary package. I'll change them back. :smiley_cat: